api/controller/health.go

```diff
@@ -13,11 +13,12 @@ func GetNotifierState(database moira.Database) (*dto.NotifierState, *api.ErrorRe
 		return nil, api.ErrorInternalServer(err)
 	}
 
-	notifierState := dto.NotifierState{State: state}
-	if state == moira.SelfStateERROR {
-		notifierState.Message = dto.ErrorMessage
+	message, err  := database.GetNotifierMessage()
+	if err != nil {
```

Comment:
Why `if state == moira.SelfStateERROR` is disappeared here?

Reply:
`notifierState.Message = dto.ErrorMessage` was setting the notifier message automatically, but now, it fetches the message from the database instead.